cypress/support/utils.ts

```diff
@@ -149,26 +133,26 @@ export const assertCodePaneContains = (text) => {
   });
 };
 
-export const assertCodePaneLineCount = (lines, { wait } = {}) => {
+export const assertCodePaneLineCount = (lines: number, wait?: number) => {
   getCodeEditor().within(() =>
     cy.get('.CodeMirror-line').should('have.length', lines)
   );
 
   // Wait after check to ensure original focus is restored
-  if (typeof wait === 'number') {
+  if (wait) {
```

Comment:
This will have a behaviour change for zero. Not sure if `cy.wait(0)` pushes to next tick or not.

Reply:
Good point. Thinking now, it's probably better to always have the wait as `500` anyway, since that's the only wait time we use